src/features/venue/components/VideoSection/FakeVideoPlayer.tsx

```diff
@@ -22,7 +22,7 @@ export const FakeVideoPlayer: FunctionComponent<FakeVideoPlayerProps> = ({
   onPress,
 }) => {
   return (
-    <TouchableOpacity onPress={onPress}>
+    <TouchableOpacity onPress={onPress} delayPressIn={70} accessibilityLabel="Faux player video">
```

Comment:
Pour l'accessibilité label, peut être demander à Henri ou Mouna s'ils veulent un truc plus précis, sinon peut être @Lucasbeneston si tu as une recommandation ?

Reply:
Player est un mot anglais, il faut que ce soit en français et explicite !

Reply:
Got it, c'est modifié.